python/cudf/cudf/utils/dtypes.py

```diff
@@ -62,6 +62,8 @@
     np.dtype("int16"): pd.Int16Dtype(),
     np.dtype("int32"): pd.Int32Dtype(),
     np.dtype("int64"): pd.Int64Dtype(),
+    np.dtype("float32"): pd.Float32Dtype(),
```

Comment:
Will need to do this inverse addition in `pandas_dtypes_to_cudf_dtypes` too..

Reply:
Never mind, we don't need this as it's being handled below in line 97 with proper versioning.:
```python
if PANDAS_GE_120:
    cudf_dtypes_to_pandas_dtypes[np.dtype("float32")] = pd.Float32Dtype()
    cudf_dtypes_to_pandas_dtypes[np.dtype("float64")] = pd.Float64Dtype()
    pandas_dtypes_to_cudf_dtypes[pd.Float32Dtype()] = np.dtype("float32")
    pandas_dtypes_to_cudf_dtypes[pd.Float64Dtype()] = np.dtype("float64")
```